src/nvim/api/vim.c

```diff
@@ -53,6 +54,14 @@
 # include "api/vim.c.generated.h"
 #endif
 
+void nvim_source(String command, Error *err)
+  FUNC_API_SINCE(6)
+{
+    try_start();
+    do_source_str((char_u *)command.data);
+    try_end(err);
```

Comment:
`nvim_source` should live next to `nvim_command_output` perhaps. At least not before macros.

Also, 2-space indent.